experimental/traffic-portal/src/app/core/cdns/cdn-table/cdn-table.component.ts

```diff
@@ -0,0 +1,276 @@
+			href: (selectedRow): string => `/core/cdns/${selectedRow.id}/dnssec-keys`,
+			name: "Manage DNSSEC Keys"
+		},
+		{
+			disabled: (): true => true,
+			href: (selectedRow): string => `/core/cdns/${selectedRow.id}/federations`,
+			name: "Manage Federations"
+		},
+		{
+			disabled: (): true => true,
+			href: (selectedRow): string => `/core/cdns/${selectedRow.id}/delivery-services`,
+			name: "Manage Delivery Services"
+		},
+		{
+			href: "/core/profiles",
+			name: "View Profiles",
+			queryParams: (selectedRow): Params => ({cdnName: selectedRow.name}),
+		},
+		{
+			href: "/core/servers",
+			name: "View Servers",
+			queryParams: (selectedRow): Params => ({cdnName: selectedRow.name}),
+		},
+		{
+			disabled: (): true => true,
+			href: (selectedRow): string => `/core/cdns/${selectedRow.id}/notifications`,
+			name: "Manage Notifications"
+		},
+	];
+
+	/**
+	 * A subject that child components can subscribe to for access to the fuzzy
+	 * search query text.
+	 */
+	public fuzzySubject: BehaviorSubject<string>;
+
+	/** Form controller for the user search input. */
+	public fuzzControl: FormControl = new FormControl("");
+
+	constructor(
+		private readonly alerts: AlertService,
+		private readonly api: CDNService,
+		public readonly auth: CurrentUserService,
+		private readonly dialog: MatDialog,
+		private readonly route: ActivatedRoute,
+	) {
+		this.fuzzySubject = new BehaviorSubject<string>("");
+		this.cdns = this.api.getCDNs();
+	}
+
+	/** Initializes table data, loading it from Traffic Ops. */
+	public ngOnInit(): void {
+		this.route.queryParamMap.subscribe(
+			m => {
+				const search = m.get("search");
+				if (search) {
+					this.fuzzControl.setValue(decodeURIComponent(search));
+					this.updateURL();
+				}
+			}
+		);
+	}
+
+	/** Update the URL's 'search' query parameter for the user's search input. */
+	public updateURL(): void {
+		this.fuzzySubject.next(this.fuzzControl.value);
+	}
+
+	/**
+	 * Queues or clears updates on a group of CDNs.
+	 *
+	 * @param cdn The CDN on which to queue updates.
+	 * @param queue Whether updates should be queued (`true`) or cleared
+	 * (`false`).
+	 */
+	private async queueUpdates(cdn: ResponseCDN | number, queue: boolean = true): Promise<void> {
+		if (typeof cdn === "number") {
+			cdn = await this.api.getCDNs(cdn);
+		}
+		const readableAction = queue ? "Queue" : "Clear";
+		const title = `${readableAction} Updates on ${cdn.name}?`;
+		const ref = this.dialog.open<DecisionDialogComponent, DecisionDialogData, boolean>(DecisionDialogComponent, {
+			data: {
+				message: `Are you sure you want to ${readableAction.toLowerCase()} server updates for all of the ${cdn.name} servers?`,
+				title,
+			}
+		});
+		if (!await ref.afterClosed().toPromise()) {
+			return;
+		}
+		if (queue) {
+			await this.api.queueServerUpdates(cdn);
+		} else {
+			await this.api.dequeueServerUpdates(cdn);
+		}
+		this.alerts.newAlert(
+			AlertLevel.SUCCESS,
+			`${readableAction}d CDN server updates`,
```

Comment:
When updates are cleared, this will show an alert reading "Cleard CDN server updates". Should be "Clear<ins>e</ins>d CDN server updates".

Reply:
Alert reads "Clear<ins>e</ins>d CDN server updates" in 409c6170a7